.all-contributorsrc

```diff
@@ -7700,6 +7700,15 @@
         "bug"
       ]
     },
+    {
+      "login": "dschach",
+      "name": "David Schach",
+      "avatar_url": "https://avatars.githubusercontent.com/u/636977?v=4",
+      "profile": "https://www.x2od.com/",
```

Comment:
Can this be changed to https://github.com/dschach

Reply:
I've updated it. The initial value is taken from your github profile, where you have added this webpage...